services/mailer.js

```diff
@@ -0,0 +1,22 @@
+require('dotenv').config()
+
+const sendConfirmationEmail = () => {
+    const mailgun = require('mailgun-js');
+    const DOMAIN = process.env.DOMAIN;
+    const mg = mailgun({apiKey: process.env.MAILGUN_API_KEY, domain: process.env.DOMAIN});
```

Comment:
you don't want to initialize everytime the function is run. You want this outside the function so it gets initialized before the function is run. This function just sends the email.